App/Lettuce/Tests/LettuceTests/DownloadAttachmentRequestTests.swift

```diff
@@ -0,0 +1,17 @@
+import XCTest
+@testable import Lettuce
+
+final class DownloadAttachmentRequestTests: XCTestCase {
+
+    func test() throws {
```

Comment:
Check `WriteNewMessageRequestTests` for inspiration on what to test. Sample JSON is taken from API documentation.